metricbeat/module/system/swap/swap.go

```diff
@@ -0,0 +1,47 @@
+// +build darwin freebsd linux openbsd windows
```

Comment:
Is this really implemented for Windows or does this always just return error events? If not implemented, remove `windows`.